README.rst

```diff
@@ -1,4 +1,20 @@
 Job Output Folder Data Type Plugin
 ==================================
 
-This is the data type plugin for Qiita jobs that only create a folder and the command is responsible of generating all the files.
+The goal of this Qiita type plugin is to validate and summarize any kind of folder output.
+
+Note that `job-output-folder` expects a single folder and this will become an artifact that will live in
+`[qiita-base-path]/job-output-folder/[artifact-id]/[output-folder]` and this plugin will generate:
+
+- `summary.html`: a browser friendly file listing that will include all files at `[artifact-id]/[output-folder]` and
+  any `index.html` files in any subfolder. As a reminder, the Qiita nginx basic configuration allows to display/load any
+  html/JS available files; thus, able to display properly `index.html` files available
+- `MANIFEST.txt`: a comprehensive list of all available files in the folder.
```

Comment:
from the wording, it is not 100% clear to me if 
- "will include all files at `[artifact-id]/[output-folder]`" includes recursion through all sub-directories
- "Qiita nginx basic configuration allows to display/load any html/JS available files" which lines in the example configuration refers to this? Can you point to those?
- which class of user can make use of the `MANIFEST.txt`. Can he/she download the whole directory as a ZIP archive?